static/app/routes.tsx

```diff
@@ -1004,6 +1004,11 @@ function buildRoutes() {
           componentPromise={() => import('sentry/views/dashboardsV2/widgetBuilder')}
           component={SafeLazyLoad}
         />
+        <Route
+          path=":widgetId/"
```

Comment:
I think the path should be "widget/:widgetId/"

Reply:
do we need a new route? could we maybe reuse the` widget/:widgetId/edit/` route?

Reply:
`widget/:widgetId/` sounds good. I think I want to avoid using `widget/:widgetId/edit/` since the widget viewer isn't really related to editing. I've made the update to `widget/:widgetId/`!

Reply:
oh yes sure 😄